public/app/features/explore/state/query.test.ts

```diff
@@ -80,6 +85,9 @@ jest.mock('@grafana/runtime', () => ({
       get: (uid?: string) => datasources.find((ds) => ds.uid === uid) || datasources[0],
     };
   },
+  config: {
+    featureToggles: () => useFeatToggles,
```

Comment:
Hmm... `featureToggles` is an object not a function. Shouldn't it be:

```suggestion
    featureToggles: useFeatToggles,
```